server/sdk/types/ITeamService.ts

```diff
@@ -64,7 +65,7 @@ export interface ITeamService {
 	listByNames(names: Array<string>, options?: FindOneOptions<ITeam>): Promise<Array<ITeam>>;
 	listByIds(ids: Array<string>, options?: FindOneOptions<ITeam>): Promise<ITeam[]>;
 	search(userId: string, term: string | RegExp, options?: FindOneOptions<ITeam>): Promise<ITeam[]>;
-	members(uid: string, teamId: string, canSeeAll: boolean, options?: IPaginationOptions, queryOptions?: IQueryOptions<ITeamMember>): Promise<IRecordsWithTotal<ITeamMemberInfo>>;
+	members(uid: string, teamId: string, canSeeAll: boolean, options?: IPaginationOptions, queryOptions?: Partial<FindOneOptions<IUser>>): Promise<IRecordsWithTotal<ITeamMemberInfo>>;
```

Comment:
I don't think you need `Partial` for `FindOneOptions` 🤔 since all keys of the type are already optional